jbpm-db-scripts/src/main/resources/db/upgrade-scripts/mysql5/jbpm-7.59-to-7.60.sql

```diff
@@ -3,3 +3,4 @@ alter table BAMTaskSummary drop column end_date;
 alter table TaskEvent drop column end_date;
 alter table NodeInstanceLog drop column end_date;
 alter table VariableInstanceLog drop column end_date;
+alter table NodeInstanceLog add column observation varchar(255);
```

Comment:
"observation" field in the create script for mysql5 is missing

Reply:
added